src/models/streaming_server.rs

```diff
@@ -588,7 +590,8 @@ fn get_torrent_statistics<E: Env + 'static>(url: &Url, request: &StatisticsReque
         .body(())
         .expect("request builder failed");
     EffectFuture::Concurrent(
-        E::fetch::<_, Statistics>(request)
+        // `null`` can be returned when the stream has been loaded 100%
```

Comment:
It's happening when the engine is destroyed for inactivity:
https://github.com/Stremio/enginefs/blob/8722f2ecf423fc98d529aff3d0db5864e501ba04/enginefs.js#L309
If it was downloaded to 100% and that the stream is paused, then played, it will create a new engine and return the correct stats

Reply:
Fixed the comment.